docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+[//]: # (title: Swift Package export setup)
+
+Kotlin/Native output for an Apple target can be consumed as an SPM dependency.
+For example, if your Kotlin Multiplatform project has an iOS target, you may want to make the binary available
+as a dependency to iOS developers working on native Swift projects in Xcode. Using KMP tooling, you can provide
+an artifact that would seamlessly integrate with Xcode experience.
+
+This guide shows how to do this using the [XCFramework building](multiplatform-build-native-binaries.md#build-xcframeworks)
+capability of Kotlin Gradle.
+
+## Prepare locations for the files to be uploaded
+
+To make your framework available to be consumed, you will need to upload two files:
+* A ZIP archive of the XCFramework itself should be uploaded to a convenient file storage with direct access (for example,
+S3, GitHub Releases, or a Maven repository). Choose the option that is easiest to integrate into your workflow.
+* The `Package.swift` file describing the package needs to be placed in a Git repository. You need to decide which
+repository to use. Pros and cons for several options are discussed below.
+
+### Options for Package.swift distribution
+The `Package.swift` that describes the Swift Package needs to be placed in a Git repository:
+* Store the `Package.swift` file in an independent repository to make its versioning separate from the
+KMP project the file describes. This is the recommended approach, allowing scalability and generally easier
+maintenance.
+* Put the `Package.swift` file next to your KMP code for a more straightforward approach. Keep in mind that versioning
+a Swift Package in the same repository as the code can become challenging as SPM uses Git tags for versioning packages
+and this can conflict with using tags for the project itself. 
+* Store the `Package.swift` within the consumer project's repo to avoid the versioning and maintenance issues.
+
+  However, this approach can cause problems for multi-repository SPM setups of the consumer project and further automation:
+  1. Only one consumer package in a multi-package project can depend on the external module, to avoid dependency conflicts
+  within the project. So all the logic depending on your KMP module should be encapsulated in a particular consumer package.
+  2. If you publish the KMP project using an automated CI process, this process would need to include publishing the
+  updated `Package.swift` file to the consumer repo. Such a phase in CI can be difficult to maintain as it may lead
+  to conflicting updates of the consumer repo.
+
+Now that you understand the options, create or adjust a Git repository as needed.
```

Comment:
I suggest removing this sentence. It can be confusing as it seems to introduce the next section, but is actually related to the Prepare locations section